[docs] add note for update/delete using secondary sync index (#390)
ydb-platform/ydb: ydb/docs/en/core/dev/secondary-indexes.md

UPDATE table1 ON SELECT * FROM $to_update
```

{% note info %}

> Currently, data updating is possible only using a synchronous secondary index. This is because data modification is possible only in [Serializable](../concepts/transactions.md#modes) transactions, but accessing asynchronous indices would break the guarantees of this transaction mode.

Reviewer: ```suggestion
Currently, data updating is possible only using a synchronous secondary index. This limitation exists because data modification is permitted only in [Serializable](../concepts/transactions.md#modes) transactions, and accessing asynchronous indices would violate the guarantees of this transaction mode.
```

WHERE views = 0;
```

{% note info %}

> Currently, deleting data is possible only using a synchronous secondary index. This is because data removal is possible only in [Serializable](../concepts/transactions.md#modes) transactions, but accessing asynchronous indices would break the guarantees of this transaction mode.

Reviewer: ```suggestion
Currently, deleting data is possible only using a synchronous secondary index. This is because data removal is permitted only in [Serializable](../concepts/transactions.md#modes) transactions, and accessing asynchronous indices would violate the guarantees of this transaction mode.
```
